core/build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'kotlin-android-extensions'
 ext {
     PUBLISH_GROUP_ID = 'com.arcrobotics'
     PUBLISH_ARTIFACT_ID = 'ftclib'
-    PUBLISH_VERSION = '1.1.2'
+    PUBLISH_VERSION = '1.1.3'
```

Comment:
Why are we bumping the version to 1.1.3 here, but also having a bump to 1.2.0 in #114 and everywhere else here?

Reply:
This version bump was caused by pulling changes in my local from dev.